terraform/20-app/cloud-front.front-end.tf

```diff
@@ -252,3 +252,48 @@ module "cloudfront_password_protection_frontend" {
   create = local.add_password_protection
   name   = "${local.prefix}-front-end-password-protection"
 }
+
+
+################################################################################
+# Response policies
+################################################################################
+resource "aws_cloudfront_response_headers_policy" "front_end" {
+  name = "${local.prefix}-front-end"
+
+  cors_config {
+    access_control_allow_credentials = true
+    access_control_allow_headers {
+      items = ["*"]
+    }
+    access_control_allow_origins {
+      items = ["*"]
+    }
+    access_control_allow_methods {
+      items = ["GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
+    }
+    origin_override = true
+  }
+
+  security_headers_config {
+    referrer_policy {
+      referrer_policy = "strict-origin-when-cross-origin"
+      override        = false
+    }
+    strict_transport_security {
+      access_control_max_age_sec = 31536000
+      override                   = false
+    }
+    content_type_options {
+      override = false
+    }
+    frame_options {
+      frame_option = "DENY"
```

Comment:
The managed policy we were previously using set this to `SAMEORIGIN`
But we want `DENY`